### DISCUSSION TITLE
Error 500 on Vendor Model report

### DISCUSSION THREAD

**Discussion by calvaradESU** [opendcim/openDCIM#1443] [Q&A] [answered]
When I try to run a Vendor Model report I get an error 500, and this in apache log:

[Fri Sep 15 15:02:11.051322 2023] [php:error] [pid 1043189] [client 10.222.24.44:50173] PHP Fatal error:  Uncaught Error: Undefined constant PhpOffice\\PhpSpreadsheet\\Collection\\CellsFactory::cache_in_memory_serialized in /var/www/openDCIM-23.02/report_vendor_model.php:96\nStack trace:\n#0 {main}\n  thrown in /var/www/openDCIM-23.02/report_vendor_model.php on line 96, referer: https://dcim.emporia.edu/report_vendor_model.php

This also happens on the demo site.

**Comment by gerases** [comment, marked as answer]
Commenting out these two lines fixes it:

```
$cacheMethod = \PhpOffice\PhpSpreadsheet\Collection\CellsFactory::cache_in_memory_serialized;
$retcode = \PhpOffice\PhpSpreadsheet\Settings::setCacheStorageMethod($cacheMethod);
```
